### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by BriaWilliams**
Pull this in dangit

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ cron.schedule('0 0 9 * * *', () => {
         console.log(date);
         console.log(title);
         console.log(explanation);
-        const sms = `Hey babe,\nhere is the NASA image for ${formateDate(date)}\n\n${url}\n\nTitle: ${title}\n\nContext: ${explanation}\n\n`;
+        const sms = `Good morning Jellybean,\nhere is the NASA image for ${formateDate(date)}\n\n${url}\n\nTitle: ${title}\n\nContext: ${explanation}\n\n`;
```
Comment (juniorro) [inline src/index.js:35]: I didn't say change this 🙃🙃

File: src/index.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ const sendEmail = async (message) =>  {
     }
   });
   const response = await transporter.sendMail({
-    from: '"👻" <>',
+    from: '"Zaddy" <>',
```
Comment (juniorro) [inline src/index.js:57]: Nor this 🧐🧐

**Review by juniorro — APPROVED** [review]
Good job

File: src/index.js (hunk 1 above)
Comment (BriaWilliams) [reply src/index.js:35]: I do what I want.

**Pull request merged**